pkgs/applications/science/biology/dcm2niix/default.nix

```diff
@@ -1,10 +1,24 @@
-{ lib, stdenv
+{ lib
+, stdenv
 , fetchFromGitHub
+, substituteAll
 , cmake
+, openjpeg
 , libyamlcpp
-, git
+, batchVersion ? false
+, withJpegLs ? true
+, withOpenJpeg ? true
+, withCloudflareZlib ? true
 }:
 
+let
+  cloudflareZlib = fetchFromGitHub {
+    owner = "ningfei";
+    repo = "zlib";
+    rev = "gcc.amd64";
```

Comment:
We can't track a branch. As soon as a commit is pushed there the hash is wrong. Please track a commit or tag.

Reply:
> We can't track a branch.

Is this a Nixpkgs contribution policy? 

> As soon as a commit is pushed there the hash is wrong.

Yes, I know this. However, the upstream project also pulls the latest commit from the `gcc.amd64` branch. So if there is an update to that branch, I want the Nix package build to fail as a prompt to update the hash. That way, we'll stay in sync with the upstream version of the dependency.

Before I proceed with any changes, please share your thoughts on this @SuperSandro2000.

Reply:
> Is this a Nixpkgs contribution policy?

I don't know but as soon as someone pushes to that branch the has is wrong which is not a reproducible build in any way.

> I want the Nix package build to fail as a prompt to update the hash.

Thats not how packages are supposed to work. We use update scripts and bots for this.

Reply:
Please take another look @SuperSandro2000.

Reply:
Hey @SuperSandro2000, this has been hanging out for a long time, can you take a look or remove yourself as a reviewer?